setup.py

```diff
@@ -122,13 +122,13 @@ def get_version(version_file):
         "psutil",
         "psycopg2-binary == 2.9.3",
         "pymongo == 4.0.1",
-        "pymupdf == 1.19.0",
+        "pymupdf",
         "python-dateutil >= 2.7.3",
         "pytest-cov",
         "python-pptx == 0.6.21",
         "pytz",
         "pyyaml == 6.0",
-        "reportlab == 3.6.6",
+        "reportlab",
```

Comment:
The PR description should mention that you are removing these version pins and why you are doing so.

Reply:
I have reverted these changes. It originally looked as though these pinned packages were causing problems.

Reply:
Reverted in https://github.com/cisagov/pe-reports/pull/427/commits/1c8b9b49bf1d187d28773c186d4dd4c91aa59b87.